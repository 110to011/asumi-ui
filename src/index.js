/**
 * Created by Elly on 2016/5/27.
 */

import Menu from './component/menu';
import SubMenu from './component/menu/subMenu';
import MenuItem from './component/menu/menuItem';
import MenuItemGroup from './component/menu/menuItemGroup';
import Tag from './component/tag';
import Grid from './component/grid';
import Form from './component/form';
import Tabs from './component/tabs';
import Radio from './component/radio';
import Group from './component/group';
import Input from './component/input';
import Modal from './component/modal';
import Table from './component/table';
import Editor from './component/editor';
import Col from './component/table/col';
import Button from './component/button';
import Select from './component/select';
import Loading from './component/loading';
import Popover from './component/popover';
import Message from './component/message';
import Animate from './component/animate';
import Upload from './component/upload';
import Checkbox from './component/checkbox';
import Dropdown from './component/dropdown';
import Option from './component/select/option';
import Pagination from './component/pagination';
import FormItem from './component/form/formItem';
import TabPanel from './component/tabs/tabPanel';
import RadioGroup from './component/radio/radioGroup';
import CheckboxGroup from './component/checkbox/checkGroup';
import SimplePagination from './component/pagination/simplePagination';
import DateTime from './component/datetime';

const Tooltip = Popover;

Table.Col = Col;

Select.Option = Option;

Tabs.TabPanel = TabPanel;

Menu.SubMenu = SubMenu;
Menu.MenuItem = MenuItem;
Menu.MenuItemGroup = MenuItemGroup;

Radio.Group = RadioGroup;
Checkbox.Group = CheckboxGroup;

Form.FormItem = FormItem;

Pagination.Simple = SimplePagination;

export {
    Col,
    Tag,
    Tabs,
    Menu,
    Form,
    Grid,
    Table,
    Modal,
    Input,
    Group,
    Radio,
    Select,
    Button,
    Upload,
    Option,
    Editor,
    SubMenu,
    Animate,
    Loading,
    Popover,
    Message,
    Tooltip,
    FormItem,
    MenuItem,
    TabPanel,
    Dropdown,
    Checkbox,
    DateTime,
    Pagination,
    RadioGroup,
    CheckboxGroup,
    MenuItemGroup,
    SimplePagination,
};