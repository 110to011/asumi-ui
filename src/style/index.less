@charset "utf-8";

@import "variables";
@import "normalize";
@import "fontawesome/less/font-awesome";

body {
  font-family: '微软雅黑', "Microsoft YaHei", Arial, Microsoft YaHei, Helvetica, sans-serif;
  margin: 0;
  color: @normalText;
  font-size: @fontSize;
  line-height: 1.42857;
}

a {
  color: @primary;
  &:hover, &:active, &:visited, &:focus {
    color: @d_primary;
  }
}

.el-shadow {
  box-shadow: 0 6px 12px @shadow;
  -webkit-box-shadow: 0 6px 12px @shadow;
}

.overflow {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

@import "nav";
@import "tabs";
@import "drop";
@import "caret";
@import "modal";
@import "input";
@import "table";
@import "button";
@import "stretch";
@import "popover";
@import "checkbox";
@import "pagination";
