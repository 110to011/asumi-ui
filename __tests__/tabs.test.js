import React from 'react';
import {mount} from 'enzyme';
import {Tabs, TabPanel} from '../src';

describe('Tabs', () => {
    it('create component', () => {
        const component = mount(<Tabs><TabPanel key="1">a</TabPanel></Tabs>);
        expect(component).toMatchSnapshot();
    });
});
