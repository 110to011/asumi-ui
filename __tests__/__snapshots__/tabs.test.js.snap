exports[`Tabs create component 1`] = `
<div className={"el-tabs"}>
    <ul className={`el-tabs-nav$ clearfix`}>
        <li
            key={"1"}
            onClick={[Function]}
            className={"el-tabs-nav-active"}>
            <a href="javascript:;">

            </a>
        </li>
    </ul>
    <div className={`el-tabs-content`}>
        <TabPanel key="1" _active={true}>a</TabPanel>
    </div>
</div>
`;